introduction.md

```diff
@@ -0,0 +1 @@
+Bla bla bla, here I am, bla bla bla. I am a _____ working in ______ for ______ years, and interested in ______, _______, and ______. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.